tutorials/hoon/egg-timer.md

```diff
@@ -31,7 +31,7 @@ The first thing to notice is that we are creating a `core` (`|%`) and a `door` (
 
 ```hoon
 |%
-+$  card   (pair bone move)
++$  card  (pair bone move)
 +$  move  [%wait path @da]
```

Comment:
Okay right this is just backwards and I think the source of my confusion

Reply:
Oh god I touched this and didn't even notice. Hah, thanks for the heads up!

Reply:
At least that answers the question of which parts of the code would and wouldn't compile!